security/src/main/java/io/micronaut/security/endpoints/ControllerConfiguration.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.security.endpoints;
+
+import io.micronaut.core.util.Toggleable;
+
+/**
+ * Base configuration for all controllers.
+ *
+ * @author Álvaro Sánchez-Mariscal
+ */
+public interface ControllerConfiguration extends Toggleable {
+
+    /**
+     * @return the path where the controller is enabled.
+     */
+    String getPath();
```

Comment:
annotate with nullability annotation

Reply:
Neither `KeysControllerConfiguration` nor `KeysControllerConfigurationProperties` have `getPath()` with nullability annotations. Is it null or non-null?

Reply:
Ok I see it has a default value. Adding `@NonNull`